R7RSSmallErrata.md

```diff
@@ -151,3 +151,9 @@ apply only to *non-ASCII* characters; the valid ASCII characters are given expli
 31\. In section 1.1, the sentence "Each use of a variable is associated with a lexically apparent binding of that variable" appears. However, this is only true after macroexpansion, as the binding may be hidden in a user-written macro.  Prefix the sentence with "After macros are expanded".
  
 32\. In the third paragraph of the definition of `null-environment`, the reference to `scheme-report-environment` should be to `null-environment`, and is changed accordingly.
+
+33\. In the definition of `car-internal` in the formal semantics, the &#969; parameter should be dropped.
```

Comment:
Although the formal semantics was updated to support dynamic-wind, it does not include exception handling (arguably derivable) introduced in R7RS.  If we were to later add this, it might be more correct to pass omega through to `car-internal`, in consideration of implementations which signal an error here (not required).

Reply:
In that case, the type of `car-internal` will need a fix instead.